packages/cogify/src/cogify/cli/cli.cover.ts

```diff
@@ -62,14 +62,24 @@ export const BasemapsCogifyCoverCommand = command({
       defaultValue: () => false,
       defaultValueIsSerializable: true,
     }),
+    name: option({
+      type: optional(string),
+      long: 'name',
```

Comment:
I think we want to remove this `name` parameter, and map the name in standardised workflow.